app/src/main/java/com/example/poste/http/EditAccountRequest.java

```diff
@@ -0,0 +1,11 @@
+package com.example.poste.http;
+
+public class EditAccountRequest {
+    private String oldPassword;
+    private String newPassword;
+
+    public EditAccountRequest(String oldPassword, String newPassword) {
```

Comment:
I'm wondering if we should maybe rename this to be more precise, or alternatively allow passing a key-value listing such that we can use this same endpoint for editing different account fields in the future?